check_api/src/main/java/com/google/errorprone/fixes/SuggestedFixes.java

```diff
@@ -1023,11 +1024,17 @@ public CharSequence getCharContent(boolean ignoreEncodingErrors) throws IOExcept
     Options originalOptions = Options.instance(javacTask.getContext());
     for (String key : originalOptions.keySet()) {
       String value = originalOptions.get(key);
-      if (key.equals("-Xplugin:") && value.startsWith("ErrorProne")) {
+      if (key.equals(Option.PLUGIN.getPrimaryName()) && value.startsWith("ErrorProne")) {
         // When using the -Xplugin Error Prone integration, disable Error Prone for speculative
         // recompiles to avoid infinite recursion.
         continue;
       }
+      if ((key.equals(Option.SOURCE.getPrimaryName()) || key.equals(Option.TARGET.getPrimaryName()))
+          && originalOptions.isSet(Option.RELEASE)) {
```

Comment:
This code can be optimized by factoring out `originalOptions.isSet(Option.RELEASE)` and creating a set containing `Option.SOURCE.getPrimaryName()` and `Option.TARGET.getPrimaryName()`. Decided against this because I figured it'd make this already-very-long method even more complex to follow. Let me know if you feel differently and I'll change it.